test/metabase/api/action_test.clj

```diff
@@ -305,3 +305,58 @@
           (testing "Should not have committed any of the valid rows"
             (is (= 75
                    (categories-row-count)))))))))
+
+(deftest bulk-delete-happy-path-test
+  (testing "POST /api/action/bulk/delete/:table-id"
+    (mt/test-drivers (mt/normal-drivers-with-feature :actions)
+      (actions.test-util/with-actions-test-data-and-actions-enabled
+        (is (= 75
+               (categories-row-count)))
+        (is (= {:created-rows [{(format-field-name :id) 76, (format-field-name :name) "NEW_A"}
+                               {(format-field-name :id) 77, (format-field-name :name) "NEW_B"}]}
+               (mt/user-http-request :crowberto :post 200
+                                     (format "action/bulk/create/%d" (mt/id :categories))
+                                     [{(format-field-name :name) "NEW_A"}
+                                      {(format-field-name :name) "NEW_B"}])))
+        (is (= 77
+               (categories-row-count)))
+        (is (= {:success true}
+               (mt/user-http-request :crowberto :post 200
+                                     (format "action/bulk/delete/%d" (mt/id :categories))
+                                     [{"ID" 76}
+                                      {"ID" 77}])))
+        (is (= 75
+               (categories-row-count)))))))
+
+(deftest bulk-delete-failure-test
+  (testing "POST /api/action/bulk/delete/:table-id"
+    (mt/test-drivers (mt/normal-drivers-with-feature :actions)
+      (actions.test-util/with-actions-test-data-and-actions-enabled
+        (mt/test-drivers (mt/normal-drivers-with-feature :actions)
+          (actions.test-util/with-actions-test-data-and-actions-enabled
+            (testing "error in some of the rows"
+              (is (= 75
+                     (categories-row-count)))
+              (is (= {:created-rows [{(format-field-name :id) 76, (format-field-name :name) "NEW_A"}
+                                     {(format-field-name :id) 77, (format-field-name :name) "NEW_B"}]}
+                     (mt/user-http-request :crowberto :post 200
+                                           (format "action/bulk/create/%d" (mt/id :categories))
+                                           [{(format-field-name :name) "NEW_A"}
+                                            {(format-field-name :name) "NEW_B"}])))
+              (is (= 77
+                     (categories-row-count)))
+
```

Comment:
```suggestion
```

Same thing -- you don't need to create new rows here and even if you did need to it would be better to do it directly with `jdbc/execute!` or something like that so you don't end up testing `bulk/create` inside of a `bulk/delete` test